src/auth.ts

```
@@ -22,7 +23,12 @@ import { USER_ROLES, WAITLIST_APPROVED_CODE } from "~/server/_utils/roles";
  */
 export const authConfig = {
   adapter: PrismaAdapter(db) as Adapter,
-  providers,
+  providers: [
+    ...providers,
+    ...(env.RESEND_API_KEY
+      ? [ResendProvider({ from: env.EMAIL_FROM, apiKey: env.RESEND_API_KEY })]
```

ESLint

[error] Expected object keys to be in natural insensitive ascending order. 'apiKey' should be before 'from'.

(sort-keys-fix/sort-keys-fix)